.github/ISSUE_TEMPLATE/payout-request-proposal.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Payout Request Proposal
+about: Raise a payout request from the Paseo bounty
+title: "[PROVIDER] - [Q#] - [2024]"
+labels: payout-request
+assignees: ''
+
+---
+
+Name: 
+
+Contact details (tg/email/element): 
+
+Proponent: 
+
+Requested DOT: 
+
+Proactive / Retroactive payment: 
+
+Description: 
+
+Evidence:
+
+you can use the payout calculator here https://docs.google.com/spreadsheets/d/10R4s5LVsxHnTbQeHVEd1YXkC09SjjQTMm0Cg9BH6lgY/edit?gid=1924769351#gid=1924769351
```

Comment:
I wouldn't add the link to this spreadsheet as it's more for internal tracking and the next year,  the process would change.

I would add a note saying the EMA 7 of the proposal to be taken into account (6.799)

This should be updated as soon as the new proposal is approved.